docs/gpu-driver/gpu-driver.rst

```diff
@@ -0,0 +1,53 @@
+MIGraphX GPU-driver
+===============
+
+The MIGraphX gpu-driver is used to test the performance of GPU kernels produced in MIGraphX.
+Example usage::
+    
+    gpu-driver kernel_test.json
+
+The input json file describes the gpu kernel and the input data settings.
+Random data is passed into the gpu kernel and the kernel is run a set number of iterations
+and timed for performance.
+
+json file formatting
+--------------------
+
+* settings:
+    * iterations: the number of iterations to run the kernel
+    * lens: the dimensions for the input data shape
+    * strides: strides for the input dimension, optional for standard shapes
+    * type: data type
+* compile_op:
+    * name: name of the gpu operation to compile
+    * lambda: lambda function
+    * inputs: input shapes into the kernel, need 1 more than lambda function input for output buffer
+
+*TODO: many other possible settings*
+
+Example gpu-driver hjson testing a pointwise GELU approximation::
```

Comment:
"Example gpu-driver hjson testing a pointwise GELU approximation": perhaps should be reworded.